firmware/boron-ins-fsm/test/mocks/Particle.h

```diff
@@ -48,4 +62,51 @@ class MockParticle
     }
 };
 extern String fullPublishString;
-extern MockParticle Particle;
\ No newline at end of file
+extern MockParticle Particle;
+
+typedef enum LogLevel {
+    LOG_LEVEL_ALL = 1, // Log all messages
+    LOG_LEVEL_TRACE = 1,
+    LOG_LEVEL_INFO = 30,
+    LOG_LEVEL_WARN = 40,
+    LOG_LEVEL_ERROR = 50,
+    LOG_LEVEL_PANIC = 60,
+    LOG_LEVEL_NONE = 70 // Do not log any messages
+} LogLevel;
+
+class MockLogger
```

Comment:
Can the mocklogger go in its own file?

Reply:
I moved it to its own mock_logger.h file in my latest commit.